compiler/test/dotc/tests.scala

```diff
@@ -153,6 +153,7 @@ class tests extends CompilerTest {
   @Test def pos_anonClassSubtyping = compileFile(posDir, "anonClassSubtyping", twice)
   @Test def pos_extmethods = compileFile(posDir, "extmethods", twice)
   @Test def pos_companions = compileFile(posDir, "companions", twice)
+  @Test def posVarargsInMethodsT1625 = compileFiles(posDir + "varargsInMethodsT1625/")
```

Comment:
This should also apply to constructors - I think your implementation supports this, I guess it's just the name that is misleading.